src/Components/List/List.css

```diff
@@ -0,0 +1,34 @@
+.emptyList {
```

Comment:
Could this be changed to `.listContainer` and used for both the empty and populated lists?